e2e/benchmarks/tasks_concurrent_test.go

```diff
@@ -67,19 +68,14 @@ func benchmarkTasksConcurrent(b *testing.B, numTasks, numServices int) {
 		}
 	})
 
-	b.Run("once mode", func(b *testing.B) {
-		// Run through once to initialize Terraform workspaces to isolate the
-		// concurrent execution to only Terraform applies.
-		for n := 0; n < b.N; n++ {
-			err = rwCtrl.Once(ctx)
-			require.NoError(b, err)
-		}
-	})
+	err = rwCtrl.Once(ctx)
+	require.NoError(b, err)
 
 	b.Run("task concurrent execution", func(b *testing.B) {
 		// This is the crux of the benchmark which evaluates the performance of
 		// tasks triggered and executing concurrently.
 		for n := 0; n < b.N; n++ {
+			ctx, ctxCancel = context.WithCancel(context.Background())
```

Comment:
the `ctxCancel` in L52 is no longer used and is overwritten by this change. Can you remove L52 and simplify the init in L66 to `err = rwCtrl.Init(context.Background())`?